spec/features/staff/users_can_edit_an_activity_spec.rb

```diff
@@ -413,6 +413,42 @@
         click_button t("form.button.activity.submit")
         expect(page).to have_content t("activerecord.errors.models.activity.attributes.planned_start_date.invalid")
       end
+
+      it "the policy markers can be added" do
+        activity = create(:project_activity, organisation: user.organisation)
+        # Report needs to exist so the activity is editable
+        _report = create(:report, state: :active, organisation: user.organisation, fund: activity.associated_fund)
+
+        visit organisation_activity_details_path(activity.organisation, activity)
+
+        within(".policy_marker_gender") do
+          click_on(t("default.link.edit"))
+          expect(page).to have_current_path(
+            activity_step_path(activity, :policy_markers)
+          )
+        end
+
+        choose "activity-policy-marker-gender-significant-objective-field"
+        click_button t("form.button.activity.submit")
+
+        activity.reload
+        expect(activity.policy_marker_gender).to eql("significant_objective")
+      end
+
+      it "the existing policy marker selections are shown and preserved on edit" do
+        activity = create(:project_activity, organisation: user.organisation, policy_marker_desertification: "significant_objective")
+        # Report needs to exist so the activity is editable
+        _report = create(:report, state: :active, organisation: user.organisation, fund: activity.associated_fund)
+
+        visit activity_step_path(activity, :policy_markers)
+
+        expect(page.find(:css, "#activity-policy-marker-desertification-significant-objective-field")).to be_checked
+
+        click_button t("form.button.activity.submit")
+        activity.reload
+
+        expect(activity.policy_marker_desertification).to eql("significant_objective")
```

Comment:
as above, maybe:
```
        expect(page).to have_css(
          ".policy_marker_desertification dd", 
          text: "Significant objective"
        )
```
would be a better assertion here?

Reply:
I'm not sure about the implementation detail of it being a `dd` - this would make the spec brittle to us changing the DOM  without really changing the meaning, whereas the class can be applied to any element so is a more durable (and semantic) assertion.